lib/scidata/imdb_reviews.ex

```diff
@@ -8,13 +8,21 @@ defmodule Scidata.IMDBReviews do
 
   alias Scidata.Utils
 
+  @type sentiments :: [atom]
+  @type transform_type :: atom
+  @type transform_fn :: ([binary, ...] -> any)
+  @type transform_opt :: {transform_type, transform_fn}
+  @type opts :: [transform_opt]
```

Comment:
I think we can explicitly document the options. Something like this:

```suggestion
  @type opts :: [
    transform_inputs: (...precisetype...),
    transform_labels: (...precisetype...)
  ]
```